packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -118,11 +116,22 @@ function getCurrentAction(prevActName) {
   return actions[prevActName]?.action
 }
 
-function getHoveredAtomId(atoms: Map<number, ReAtom>): number | null {
-  for (const [id, atom] of atoms.entries()) {
-    if (atom.hover) return id
+function getHoveredItemsIds(
+  ctab: Record<string, Map<number, Record<string, unknown>>>
+): Record<string, number[]> {
+  const hoveredItems = {}
+
+  for (const ctabItem in ctab) {
+    if (!(ctab[ctabItem] instanceof Map)) continue
```

Comment:
Well... all of them? Atoms, bonds, arrows, sgroups, whatever there is or will be in the future.
Previously this function checked all the atoms. I just expanded it to all items in general.
I don't see how I can use a Union here as long as we don't seem to have the single source of truth.

Reply:
I mean - e.g we try to find id of the hovered atom. So we'll check 3-4-5-8 - no matter how many records of atoms Map, find hovered and that's all. But as I understand this code - we'll go to the next Map and try to find there hovered element too, although as I suppose we can have only 1 hovered elem so it will be redundant iterations

BTW, as we agreed earlier, we don't use one-str if

Reply:
Added cycle break if the element is found.
Replaced 1-liner IFs with 3-liners.